app.js

```diff
@@ -2,26 +2,52 @@
 
 const express = require('express');
 const app = express();
+const gapi = require('./domains');
 
 app.set('views', __dirname + '/views');
 app.use(express.static('dist'));
 app.engine('html', require('ejs').renderFile);
 
+let baseEnvironment;
+const config = clientConfiguration();
+
 /**
  * Main view for manual entry
  */
-app.get('*', function (req, res) {
+app.get('/', function (req, res) {
```

Comment:
Good to know that order determines precedence. I thought the way routes were handled was more declarative. Will change.